include/conf/load.hpp

```diff
@@ -41,6 +41,20 @@ namespace color_coded
 
         return line;
       }
+
+      void make_absolute(std::string &line, std::string &next, const fs::path &base) {
```

Comment:
Since `make_absolute` makes a lot of assumptions about the data it's working with, with both `line` and `next` and their relationship, I think it'd be a cleaner approach to just have `make_absolute` work with the whole range. i.e.

```c++
template <typename It>
void make_absolute(It begin, It end,  fs::path const &base)
```

Reply:
I'm not sure I followed, make_absolute line/next is for the case where:
```
-I | -isystem | ...
../some/path/to/include
```

You mean there would be a case where:
```
-I | -isystem | ...
<empty line>
<empty line>
../some/path/to/include
```

Or just a single API make all directories absolute.